arm_robots/src/arm_robots/robot.py

```diff
@@ -426,6 +428,32 @@ def get_num_joints(self, group_name: Optional[str] = None):
     def send_joint_command(self, joint_names: List[str], trajectory_point: JointTrajectoryPoint) -> Tuple[bool, str]:
         raise NotImplementedError()
 
+    def display_robot_traj(self, trajectory: RobotTrajectory, label: str, color: Optional = None):
```

Comment:
I'd like an example of using display_robot_traj in the `basic_motion` scripts and `.rviz` file